### DISCUSSION TITLE
Does the DSC provide CORS support?

### DISCUSSION THREAD

**Discussion by logicalfarhad** [International-Data-Spaces-Association/DataspaceConnector#106] [Q&A]
Hi there

I am currently trying to make a web application using angular js. I am trying to consume the api's described in the swagger api documentation. My application is running on port 4200 and IDS running on port 8080. When I want to make a post request to api/offers with authentication header I am getting the following error: 

_Access to XMLHttpRequest at 'https://localhost:8080/api/offers' from origin 'http://localhost:4200' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource._

Am I missing something here? Thanks in advance :) 

Best Regards
Farhad

**Comment by AitorF** [comment]
Same here. In our case, the API is behind basic auth credentials and even the preflight requests fails with a `403`. I guess it is because the API is expecting credentials en each request and preflight requests (according to the [specs](https://fetch.spec.whatwg.org/#cors-protocol-and-credential)) can't be provided.
